.goreleaser.yaml

```diff
@@ -0,0 +1,56 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+
+# The lines below are called `modelines`. See `:help modeline`
+# Feel free to remove those if you don't want/need to use them.
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+version: 1
+projectname: go-aeneas
+
+
+env:
+  - GITHUB_TOKEN: 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
```

Comment:
You should not have GITHUB_TOKEN checked in.

Reply:
I believe that when go-releaser runs in GitHub Actions that it will automatically have GITHUB_TOKEN defined.  You can get around this locally by using the argument `--snapshot`.  You will also want to use `--rm-dist` which removes the dist folder before it builds (to do a clean build).